core/lib/mno_enterprise/platform_adapters/nex_adapter.rb

```diff
@@ -125,7 +152,7 @@ def setup_nex_client
         def nex_app
           @nex_app ||= begin
             setup_nex_client
-            NexClient::App.find(ENV['SELF_NEX_APP_ID']).first
+            NexClient::App.where(name: ENV['SELF_NEX_APP_NAME']).first
```

Comment:
I changed SELF_NEX_APP_ID to SELF_NEX_APP_NAME because filtering on exec_cmds has been setup to filter on executor's name and not id